dev/io.openliberty.xmlws.3.0.internal.tools/src/com/sun/tools/ws/wscompile/WsimportTool.java

```diff
@@ -0,0 +1,627 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) 1997-2013 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Did you pull this source from the ws tools code from the 3.0, or is this a modified version of the 2.2 version?  If the latter, maybe we should start with the 3.0 one and then make the modification in the 2.2 one so that we have the latest 3.0 code as the starting point.

Reply:
Yeah, I originally tried with 3.0 but ran into some weird compiler issues, but I took some more time with it and figured out what was going wrong with the bnd.bnd. That then required some creative tinkering with the changes we'd made to the original 2.2.10. I couldn't use `"-Xbootclasspath/p:"` which was allowing us to add the JAX-WS/JAX-B jars onto the class path, so I'd like to know if what I came up with will fly. I've marked our changes with // Liberty Start